packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -390,13 +390,12 @@ export class ProfilesUtils {
         ZoweLogger.trace("ProfilesUtils.promptAndHandleMissingCredentialManager called.");
         const header = localize(
             "ProfilesUtils.promptAndHandleMissingCredentialManager.suggestInstallHeader",
-            `Plugin of name "{0}" was defined for custom credential management on imperative.json file.`,
+            "Plugin of name '{0}' was defined for custom credential management on imperative.json file.",
```

Comment:
![image](https://github.com/zowe/vscode-extension-for-zowe/assets/37381190/21dde91f-e38a-4f48-a409-c4248a4beffe)

Reply:
Just wanted to confirm what the issue was here?

Reply:
oh, sorry, just wanted to post an updated picture 😋